src/dataset-builder/ConceptSearch.ts

```diff
@@ -30,25 +30,25 @@ type ConceptSearchProps = {
     openedConcept?: Concept
   ) => void;
   readonly actionText: string;
-  readonly datasetId: string;
+  readonly snapshotId: string;
   readonly initialCart: Concept[];
 };
 
 const DebouncedTextInput = withDebouncedChange(TextInput);
 export const ConceptSearch = (props: ConceptSearchProps) => {
-  const { initialSearch, domainOption, onCancel, onCommit, onOpenHierarchy, actionText, datasetId, initialCart } =
+  const { initialSearch, domainOption, onCancel, onCommit, onOpenHierarchy, actionText, snapshotId, initialCart } =
     props;
   const [searchText, setSearchText] = useState<string>(initialSearch);
   const [cart, setCart] = useState<Concept[]>(initialCart);
-  const [concepts, searchConcepts] = useLoadedData<GetConceptsResponse>();
+  const [concepts, enumerateConcepts] = useLoadedData<getConceptChildrenResponse>();
```

Comment:
why getConceptChildren response here?

Reply:
I think it's because they have the same response type.

Reply:
But I'd recommend changing the alias to `EnumerateConceptsResponse`?

Reply:
Enumerate and get children have the same return type. In the API code it's called `SnapshotBuilderConceptsResponse`, we could use that name here too.

Reply:
The underlying interface is named SnapshotBuilderConceptsResponse, but we're aliasing it to a more specific response name. I don't really understand the need to alias it - I'd prefer to just return SnapshotBuilderConceptsResponse. I'll test that out.